packages/menu/README.md

```diff
@@ -157,8 +319,19 @@ When `selects` is set to `inherit`, the `<sp-menu>` element will allow its `<sp-
 </sp-menu>
 ```
 
-## "change" event
+</sp-tab-panel>
+</sp-tabs>
+
+### Behaviors
+
+#### "change" event
 
 Whether `<sp-menu>` carries a selection or not, when one of the `<sp-menu-item>` children that it manages is activated the `<sp-menu>` element will dispatch a `change` event. At dispatch time, even when a selection is not held due to the absence of the `selects` attribute, the `value` of the `<sp-menu>` will correspond to the `<sp-menu-item>` that was activated. When the `selects` attribute is present, this `value` will be persisted beyond the lifecycle of the `change` event. When `selects="multiple"` the values of multiple items will be comma separated unless otherwise set via the `value-separator` attribute.
 
 Note: The `change` event is only dispatched on a left mouse click or Enter/Space keypress. Right/Middle mouse clicks will not dispatch the `change` event.
+
+### Accessibility guidelines
+
+#### Include a label
+
+In rare cases where context is sufficient and a label could be absent, make sure to have the design reviewed and approved by an accessibility expert.
```

Comment:
Not sure I can think of a reason not to have a label for a menu as it is not rendered.